platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r35/RunTasksBeforeRunActionCrossVersion.groovy

```diff
@@ -99,7 +96,6 @@ class RunTasksBeforeRunActionCrossVersion extends ToolingApiSpecification {
     }
 
     // older versions do not run any tasks
-    @ToolingApiVersion('>4.7')
```

Comment:
❓ do we really want execution with TAPI prior to 4.7?

Reply:
never mind :)

Reply:
You are right and that is exactly the point of this PR. We only want to verify the supported mode of execution in the integration tests. It does not make sense to have tests that use TAPI versions earlier than 7.0. The base test class has the `@ToolingApiVersion('>=7.0')` which is the most broad range that we want to support.